roles/sideload_kernel/files/flip_kernel

```diff
@@ -0,0 +1,166 @@
+#!/bin/bash
+
+NONRT=(kernel{-core,-modules,-modules-core,-modules-extra})
+RT=(kernel-rt{-core,-modules,-modules-core,-modules-extra})
+PREFETCH=/tmp
+
+replace_standard_kernel() {
+    local -a install
+    for pkg in "$@"; do
+        install+=(--install "${pkg}")
+    done
+    sudo rpm-ostree override remove kernel "${NONRT[@]}" "${install[@]}"
+}
+
+reinstall_kernel_packages() {
+    local -a install
+    for pkg in "$@"; do
+        install+=(--install "${pkg}")
+    done
+    sudo rpm-ostree uninstall --all "${install[@]}"
+}
+
+replaced_base_kernel() {
+    rpm-ostree status --json | jq -r '.deployments[0]."requested-base-removals"[]' | grep -q kernel-core
```

Comment:
if the script requires `jq` then add it to the requirements

Reply:
The script runs in the container (or more accurately the chroot to the host running the container), so it's not required to be present on the ansible host.

Reply:
I'll add a comment to that effect at the top of the script so it's more clear.